lib/packetgen/capture.rb

```diff
@@ -95,6 +96,11 @@ def set_options(options)
       else
         @parse = options[:parse]
       end
+      if options[:iface]
```

Comment:
#set_option is called by #initialize and #start.
So, on first call, @iface is set with 'lo', and on second call (on #start), it is set to `Pcap.lookupdev`. This is why specs failed.

You should set @iface default value in #initialize, and set @iface to options[:iface] here only if it is defined.